pkgdb/tests/test.hh

```diff
@@ -23,13 +23,16 @@
 /* -------------------------------------------------------------------------- */
 
 static const std::string nixpkgsRev
-  = "e8039594435c68eb4f780f3e9bf3972a7399c4b1";
+  = "ab5fd150146dcfe41fda501134e6503932cc8dfd";
 
 static const std::string nixpkgsRef
-  = "github:NixOS/nixpkgs/e8039594435c68eb4f780f3e9bf3972a7399c4b1";
+  = "github:NixOS/nixpkgs/ab5fd150146dcfe41fda501134e6503932cc8dfd";
```

Comment:
suggestion: reuse variable
IIRC, you should be able to concat const strings in c++

```suggestion
  = "github:NixOS/nixpkgs/" + nixpkgsRev;
```